### PR TITLE
Topnav usermenu links adjustments

### PR DESCRIPTION
**Pull request by nsharma123**
https://webarchive.jira.com/browse/WEBDEV-6587

### CODE REVIEW

File: packages/ia-topnav/src/assets/img/icon-upload-unpadded.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+export default html`
+  <svg
+    height="14"
+    width="14"
+    viewBox="8 8 24 24"
+    xmlns="http://www.w3.org/2000/svg"
+    aria-labelledby="uploadTitleID uploadDescID"
+  >
+    <title id="uploadTitleID">Upload icon</title>
+    <desc id="uploadDescID">An illustration of a horizontal line over an up pointing arrow.</desc>
```
Comment (iisa) [inline packages/ia-topnav/src/assets/img/icon-upload-unpadded.js:12]:
```suggestion
```

File: packages/ia-topnav/src/assets/img/icon-upload-unpadded.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+export default html`
+  <svg
+    height="14"
+    width="14"
+    viewBox="8 8 24 24"
+    xmlns="http://www.w3.org/2000/svg"
+    aria-labelledby="uploadTitleID uploadDescID"
+  >
+    <title id="uploadTitleID">Upload icon</title>
```
Comment (iisa) [inline packages/ia-topnav/src/assets/img/icon-upload-unpadded.js:11]:
```suggestion
    <title id="uploadTitleID">Upload files</title>
```

File: packages/ia-topnav/src/dropdown-menu.js [hunk 3]
```diff
@@ -55,9 +56,11 @@ class DropdownMenu extends TrackedElement {
     const calloutText = this.config.callouts?.[link.title];
     return html`<a
       href="${formatUrl(link.url, this.baseHost)}"
+      class="${link.class}"
```
Comment (iisa) [inline packages/ia-topnav/src/dropdown-menu.js:59]: class prop always available?

File: packages/ia-topnav/src/styles/dropdown-menu.js [hunk 4]
```diff
@@ -78,6 +78,16 @@ export default css`
     font-weight: bold;
   }
 
+  a.mobile-upload {
+    display: flex;
+    justify-content: left;
+    align-items: center;
+  }  
+  a.mobile-upload svg {
+    fill: #fff;
```
Comment (iisa) [inline packages/ia-topnav/src/styles/dropdown-menu.js:87]: ok. q: is there style based css var for this?

**Review by iisa — APPROVED** [review]
looks good, just got suggestions especially for new icon title and details info.

File: packages/ia-topnav/src/dropdown-menu.js (hunk 3 above)
Comment (nsharma123) [reply packages/ia-topnav/src/dropdown-menu.js:59]: yes, ifDefined() lib giving errors in offshoot build so keep class props always.

File: packages/ia-topnav/src/styles/dropdown-menu.js (hunk 4 above)
Comment (nsharma123) [reply packages/ia-topnav/src/styles/dropdown-menu.js]: changed with var(--white) variable. thanks!

**Pull request merged**